src/config-schema.ts

```diff
@@ -480,6 +480,120 @@ export const configSchema = {
           },
         ],
       },
+      {
+        title: 'Capture Vitals',
+        description:
+          'This application serves as an example of how a simple app can be created to repeatedly serve the same form to a user. In this scenario, a clerk at the front of the clinic is capturing vitals for patients. The clerk selects a patient with an active visit, records their vitals, and then returns to the prompt to find the next patient.',
```

Comment:
Do you have any better description in your mind @Piumal1999?
The current one is not yet suited and we will need to modify it to a better one.

Reply:
Something like this?
"Learn how to record a patient's vitals and biometric data, and review the records through the patient chart."